nautobot/extras/context_managers.py

```diff
@@ -91,22 +90,15 @@ def change_logging(change_context):
 
```

Comment:
Missing change above here to delete the `curry()` function as no longer used/necessary.